phpstan.neon

```diff
@@ -0,0 +1,4 @@
+parameters:
+    autoload_files:
+        - vendor/bin/.phpunit/phpunit-6.5/vendor/autoload.php
+        - vendor/autoload.php
```

Comment:
Missing newline at the end of file, you need to configure your phpstorm mate :P (Come to Emacs...)

Reply:
:trollface: